sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -182,6 +194,19 @@ def __init__(self, name, init_config, instances):
         self.instance_per_type_metrics = defaultdict(list)
         self.do_check = True
 
+        self.autodiscovery = is_affirmative(self.instance.get('database_autodiscovery'))
```

Comment:
```suggestion
        self.autodiscovery = is_affirmative(self.instance.get('database_autodiscovery', False))
```

Reply:
If `get` doesn't find anything, it will return `None` which should have the same effect as `False`.